Eldamarth/reactTest
# Add form functionality

#### What’s this PR do?

Creates a controlled form in a component called Form, which has a method from App called addCat, allowing a user to type in a name and an alignment in the two boxes and see a new cat box appear in the list. Restructures file tree to include directories for individual components. Also added PR template, which should automatically populate after this is merged.

#### Where should the reviewer start?

Changes have been made in App and Form, as well as in the src directory.

#### How should this be manually tested?

Should be able to add a cat box with a name and alignment after running `npm start` from the CLI and opening it on localhost

#### Any background context you want to provide?

Controlled forms are very common in React! I didn't add in placeholder text or ANY styling, as will be readily apparent, but there are comments to explain the different steps briefly 

#### What are the relevant tickets?

See below. 

#### Screenshots (if appropriate)
#### Questions:
# Implements/Fixes:
* What issue does this close?

#3
